src/screens/FeedScreen/Content.js

```diff
@@ -234,7 +242,7 @@ const Content = ({
     };
   };
   const handleHeightContainer = ({nativeEvent}) => {
-    setLayoutHeight(nativeEvent.layout.height + BUFFER_CONTENT_TEXT_HEIGHT);
+    setLayoutHeight(nativeEvent.layout.height);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing the buffer height from the layout height might affect the layout of the components. Make sure this change doesn't cause any visual issues or overlaps between components.
```diff
-    setLayoutHeight(nativeEvent.layout.height + BUFFER_CONTENT_TEXT_HEIGHT);
+    setLayoutHeight(nativeEvent.layout.height);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->